### DISCUSSION TITLE
Publish docker image

### DISCUSSION THREAD

**Discussion by mytlogos** [authorizerdev/authorizer#398] [Ideas and features]
I wanted to test out this service, but i do not see any available "official" docker image?
You do have a Dockerfile and docker-compose.yml, so i could download these and then build the image myself, but it would be even better to have a pre built docker image, that one can just download.

You could build these with github actions and upload them to docker hub or github packages?

Or did i miss an "official" docker image?

**Comment by lakhansamani** [comment]
@mytlogos this is the official image
https://hub.docker.com/r/lakhansamani/authorizer

**Reply by mytlogos** [reply]
Ah thanks for the clarification.
Can you document the usage of said image somewhere explicitly?
The docker compose file is not very...clear.

**Comment by lakhansamani** [comment]
Sure,
I will add using it with docker-compose here,
https://docs.authorizer.dev/deployment
